toqito/states/bb84.py

```diff
@@ -21,12 +21,12 @@ def bb84() -> np.ndarray:
 
     >>> from toqito.states import bb84
     >>> x = bb84()
-    >>> print(f"|0> = {x[0][0].T}, \n |1> = {x[0][1].T}")
-    >>> print(f"|+> = {x[1][0].T}, \n |-> = {x[1][1].T}")
+    >>> print(f"|0> = {x[0][0].T}, \n |1> = {x[0][1].T}")  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
```

Comment:
Hmm, same here. I'm wondering if we just need to change something to remove this ignore we're doing.

Reply:
We are not ignoring this output. `ELLIPSIS` will make sure doctest is satisfied with a substring match and `NORMALIZE_WHITESPACE` will make sure doctest doesn't fail the test when the whitespace in the expected and calculated outputs is not identical. 

![image](https://github.com/vprusso/toqito/assets/66048318/4eb00c70-324e-4c0b-8585-b40ff832d0ac)

![image](https://github.com/vprusso/toqito/assets/66048318/0f6a6bbd-eaf5-4c0e-9996-39c9dbd6a241)